src/scripts/main.js

```diff
@@ -1,9 +1,40 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  message.className = 'notification';
+  message.classList.add(type);
+
+  const messageTitle = document.createElement('h2');
+
+  messageTitle.className = 'title';
+  messageTitle.textContent = title;
+
+  const messageDescription = document.createElement('p');
+
+  messageDescription.textContent = description;
+
+  message.append(messageTitle, messageDescription);
+  document.body.append(message);
+
+  // Встановлення стилів для позиціонування
+  message.style.position = 'absolute';
+
+  message.style.left =
+    document.documentElement.clientWidth -
+    posRight -
+    message.offsetWidth +
+    'px';
```

Comment:
There is an issue with the calculation of `message.style.left`. The calculation should ensure that the notification is positioned correctly on the screen. Make sure that `message.offsetWidth` is subtracted correctly from the total width to position the notification at the correct right offset.